site/es/blog/new-in-devtools-98/index.md

```diff
@@ -30,202 +28,296 @@ draft: true
 
 <!-- ## Preview feature: Full-page accessibility tree {: #a11y-tree } -->
 
+## Función en vista previa: Árbol de accesibilidad de página completa {: #a11y-tree }
+
 <!-- The new **Full-page accessibility tree** makes it easier for you to get an overview of the full-page [accessibility tree](/blog/full-accessibility-tree/#what-is-the-accessibility-tree) and help you better understand how your web content is exposed to assistive technology.  -->
 
+El nuevo **arbol de accesibilidad de página completa** hace más fácil obtener una visión del [árbol de accesibilidad](/blog/full-accessibility-tree/#what-is-the-accessibility-tree) y te ayuda a una mejor comprensión de cómo el contenido de tu web es expuesto a la tecnología de asistencia.
+
 <!-- In the **Elements** panel, open the **Accessibility** pane and select **Enable full-page accessibility tree** checkbox. Then, reload DevTools and you will see a new accessibility button in the **Elements** panel. -->
 
+En el panel **Elementos**, abre el panel **Accessibilidad** y selecciona la casilla **Enable full-page accessibility tree**. A continuación, recarga DevTools y verás un nuevo botón de accesibilidad en el panel **Elementos**.
```

Comment:
```suggestion
En el panel **Elementos**, abre el panel **Accessibilidad** y selecciona la casilla **Activar árbol de accesibilidad de página completa**. A continuación, recarga DevTools y verás un nuevo botón de accesibilidad en el panel **Elementos**.
```

Reply:
In my Chrome Beta 98 (Language es_ES) the text appears in English, that's why I haven't made the translation. Should we change it even though users see it in English? (The same thing happens to me in some options and I decided to leave them as users would see it in their browser) What do you think?

Reply:
I see this string as "Enable full-page accessibility tree" on both "español (Latinoamérica)" and "español". I also think it makes sense to leave the string as the user reads it. @midudev could you please confirm you see the same string? 

BTW, I've seen the problem of inconsistent translations between "español (Latinoamérica)" and "español" across DevTools multiple times. Although it's unfortunate, I do think we should include the translation for both language options, since there is only one official version What's New in DevTools in Spanish.

Reply:
@and-oli I agree with you respect iclude the translation for both language.